lib/nodejs/kml-handler.js

```diff
@@ -0,0 +1,43 @@
+// kml-handler.js
+//  This file handles the sending/receiving of KML files between applications
+
+
+var net = require( "net" ),
+  fs = require( "fs" ),
+  formidable = require( "formidable" );
+
+
+
+function sendKML ( req, res, next ) {
+  var form = new formidable.IncomingForm();
+  form.keepExtensions = true;
+  // parse the file and then send response
+  form.parse( req, function( err, fields, files ) {
+    if ( !err ) {
+      var client = new net.Socket();
```

Comment:
The net module is included as part of the nodeJS core and is used to make TCP network calls.  When I tried to use socket.io-client (since we are acting as a client in this case), nodeJS complained that it couldn't fine socket.io-client.  If you want I can import the socket.io-client module and verify that everything still works after making the switch.  Let me know :)